modules/gocd/conf/cruise-config.xml

```diff
@@ -136,64 +149,36 @@
             <tasks>
               <exec command="/bin/bash" workingdir="#{SOURCE_DIR}">
                 <arg>-c</arg>
-                <arg>aws_region=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document|grep region|awk -F\" '{print $4}');
-                     if [ -d $HOME/.aws ]; then rm -rf $HOME/.aws; fi; mkdir ~/.aws;
-                       echo -e "[profile coreos-cluster] \nregion = ${aws_region} \n[default] \nregion = ${aws_region}" &gt; $HOME/.aws/config;
-                       echo -e "[coreos-cluster] \naws_access_key_id = #{AWS_ACCESS_KEY_ID}\naws_secret_access_key = #{AWS_SECRET_KEY}\n[default]\naws_access_key_id = #{AWS_ACCESS_KEY_ID}\naws_secret_access_key = #{AWS_SECRET_KEY}" &gt; $HOME/.aws/credentials;</arg>
-                <runif status="passed" />
-              </exec>
-              <exec command="/bin/bash" workingdir="#{SOURCE_DIR}">
-                <arg>-c</arg>
-                <arg>if [ ! -d "#{STAKATER_LOCATION}" ]; then sudo mkdir -p #{STAKATER_LOCATION};fi;  \
-                    if [ ! "$(ls -A #{STAKATER_LOCATION})" ]; then sudo git clone https://github.com/stakater/stakater.git #{STAKATER_LOCATION}; fi</arg>
+                <arg>if [ ! -d "/app/amibaker" ]; then sudo mkdir -p /app/amibaker;fi;  \
+                    if [ ! "$(ls -A /app/amibaker)" ]; then sudo git clone https://github.com/stakater/ami-baker.git /app/amibaker; fi</arg>
                 <runif status="passed" />
               </exec>
               <exec command="/bin/bash" workingdir="#{SOURCE_DIR}">
                 <arg>-c</arg>
-                <arg>cd #{STAKATER_LOCATION};sudo git pull origin master; sudo git fetch;sudo git checkout gocd-units;sudo git checkout -f; sudo git pull origin gocd-units;</arg>
+                <arg>cd /app/amibaker;sudo git pull origin master;</arg>
                 <runif status="passed" />
               </exec>
               <exec command="/bin/bash" workingdir="#{SOURCE_DIR}">
                 <arg>-c</arg>
-                <arg>sudo docker run -d --name terraform_${GO_PIPELINE_NAME} -v #{STAKATER_LOCATION}:/usr/src/app stakater/terraform</arg>
+                <arg>sudo docker run -d --name packer_${GO_PIPELINE_NAME} -v /app/amibaker:/usr/src/app stakater/packer</arg>
```

Comment:
@hazim1093 can't we run packer natively? or we have to run it through docker ... just a thought... no need to do rite now; as we can do it later but should create a technical debt ticket